Gemfile

```diff
@@ -2,3 +2,6 @@ source 'https://rubygems.org'
 gem 'github-pages', group: :jekyll_plugins
 gem 'jemoji'
 gem 'jekyll-octicons'
+group :jekyll_plugins do
+     gem "jekyll-polyglot"
```

Comment:
Was not aware that this plugin was available on GitHub Pages (thanks for looking into this). 🙏

Where is the list of all available ones? I thought it was https://pages.github.com/versions/, apparently not.

Reply:
Github pages runs Jekyll 3.9, so any plugins that fit with that will work. I tried a couple of different ones from this list: https://github.com/planetjekyll/awesome-jekyll-plugins#multi-language--multi-lingual but polyglot had the most support for what we want to do. I don't have strong opinions on how we do it though. 

General goal is: All of this should be replaceable, but the content should be formatted in such a way that is reusable.

Reply:
> Github pages runs Jekyll 3.9, so any plugins that fit with that will work.

I wish that was the case, but there are only a handful that are available w/ free hosting. I think there's a list somewhere.

Reply:
My comment above was inaccurate (😓), however, I was able to find the docs to what I was alluding to&hellip;

> GitHub Pages cannot build sites using unsupported plugins. If you want to use unsupported plugins, generate your site locally, and then push your site's static files to GitHub.
>
> &mdash; https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages-and-jekyll#plugins

Postmortem: seems like this was the gotcha leading to https://github.com/tc39/tc39.github.io/issues/223.
Workaround: site now built via GitHub Action w/ build product deployed to the `gh-pages` branch.